samples/highcharts/data-tools/datamodifier-tree/demo.js

```diff
@@ -0,0 +1,94 @@
+const sortModifier = new Highcharts.DataModifier.types.Sort();
```

Comment:
Error in the console.

Reply:
Works fine on my branch, did you build it?

Reply:
Yep, now works ;)